memory/src/test/java/org/opensearch/ml/memory/action/conversation/DeleteConversationTransportActionTests.java

```diff
@@ -138,8 +138,10 @@ public void testdoExecuteFails_thenFail() {
     }
 
     public void testFeatureDisabled_ThenFail() {
-        when(this.clusterService.getSettings()).thenReturn(Settings.EMPTY);
-        when(this.clusterService.getClusterSettings()).thenReturn(new ClusterSettings(Settings.EMPTY, Set.of(ConversationalIndexConstants.ML_COMMONS_MEMORY_FEATURE_ENABLED)));
+        Settings settings = Settings.builder().put(ConversationalIndexConstants.ML_COMMONS_MEMORY_FEATURE_ENABLED.getKey(), false).build();
+        when(this.clusterService.getSettings()).thenReturn(settings);
+        when(this.clusterService.getClusterSettings())
+            .thenReturn(new ClusterSettings(settings, Set.of(ConversationalIndexConstants.ML_COMMONS_MEMORY_FEATURE_ENABLED)));
```

Comment:
This line seems redundant to all the files. Can't we put this in a common file (may be testutil or something) and then invoke this.